swift/Tests/Blockchains/BandChainTests.swift

```diff
@@ -0,0 +1,405 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import TrustWalletCore
+import XCTest
+
+class BandChainTests: XCTestCase {
+    let privateKey = PrivateKey(data: Data(hexString: "1037f828ca313f4c9e120316e8e9ff25e17f07fe66ba557d5bc5e2eeb7cba8f6")!)!
+
+    func testAddress() {
+        let address = CoinType.bandChain.deriveAddress(privateKey: privateKey)
+
+        XCTAssertEqual(address, "band1jf9aaj9myrzsnmpdr7twecnaftzmku2mgms4n3")
+        XCTAssertTrue(CoinType.bandChain.validate(address: "band1pnndgfwsrff86263xzpc5cd3t6yfvgjyqc87jh"))
+        XCTAssertTrue(CoinType.bandChain.validate(address: "band1jf9aaj9myrzsnmpdr7twecnaftzmku2mgms4n3"))
+        XCTAssertFalse(CoinType.bandChain.validate(address: "cosmos1hsk6jryyqjfhp5dhc55tc9jtckygx0eph6dd02"))
+    }
+
+    func testSigningTransaction() {
```

Comment:
can you add some tx signing tests to android as well?

Reply:
@hewigovens Added tx signing tests to android as well.